origin-growth/src/apollo/resolvers.js

```diff
@@ -76,11 +77,13 @@ const resolvers = {
     }
   },
   Mutation: {
-    async invite() {
+    async invite(root, args) {
+      // FIXME: Check the referrer against Auth token.
```

Comment:
@sparrowDom My understanding is that when you implement the Terms signing, we'll then have in the DApp locale storage an Auth token that will be sent along as an HTTP header when calling any mutations on the server. This way we can check the referrer is really the one calling the mutation. Are we on the same page ?

Reply:
Yup exactly 👍

Reply:
Excellent :)